### PR TITLE
Fixed null row error

### PR DESCRIPTION
**Pull request by RoinujNosde**

### CODE REVIEW

File: app/Clans.php [hunk 1]
```diff
@@ -22,9 +23,16 @@ public function getColorTagByTag($tag = null)
         return isset($tag) ? $this->getTable()->where('tag', '=', $tag)->get('color_tag')->first() : null;
     }
 
-    public function getHTMLColorTagByTag($tag = null)
+    public function getHTMLColorTagByTag($tag = null): ?string
```
Comment (Tomut0) [inline app/Clans.php:26]: Use `getHTMLColorTagByTag($tag = null) : ?string`.
Styling.

File: app/Clans.php [hunk 2]
```diff
@@ -12,7 +13,7 @@ public function getTable()
         return DB::table('sc_clans');
     }
 
-    public function getClans()
+    public function getClans(): Collection
```
Comment (Tomut0) [inline app/Clans.php:16]: Styling.

File: app/Clans.php [hunk 3]
```diff
@@ -22,9 +23,16 @@ public function getColorTagByTag($tag = null)
         return isset($tag) ? $this->getTable()->where('tag', '=', $tag)->get('color_tag')->first() : null;
     }
 
-    public function getHTMLColorTagByTag($tag = null)
+    public function getHTMLColorTagByTag($tag = null): ?string
     {
-        return isset($tag) ? $this->addColors($this->getTable()->where('tag', '=', $tag)->get('color_tag')->first()->color_tag) : null;
+        if (!isset($tag)) {
```
Comment (Tomut0) [inline app/Clans.php:28]: _Determine if a variable is considered set, this means if a variable is declared and is different than null._ 
So, you check here: if the variable doesn't declared and is null.
How about:
```php
if (isset($tag)) {
  $row = $this->getTable()->where('tag', '=', $tag)->get('color_tag')->first();
  if (isset($row)) {
    return $this->addColors($row->color_tag);
  }
 }
 return null;
 ```
 
 Another way is:
 ```php
 if (isset($tag)) {
  $row = $this->getTable()->where('tag', '=', $tag)->get('color_tag');
  if ($row->isNotEmpty()) {
    return $this->addColors($row->first()->color_tag);
  }
 }
  return null;
 ```

File: app/Clans.php [hunk 4]
```diff
@@ -33,7 +41,8 @@ public function getHTMLColorTagByTag($tag = null)
      * @param $string - without colors.
      * @return string - with colors.
      */
-    private function addColors($string) {
+    private function addColors($string): string
```
Comment (Tomut0) [inline app/Clans.php:44]: Styling.

**Review by Tomut0 — COMMENTED** [review]
Don't forget to remove:
./idea folder's files

**Comment by RoinujNosde** [comment]
> Don't forget to remove:
> ./idea folder's files

Your repo contains this folder. 👀

File: app/Clans.php (hunk 3 above)
Comment (RoinujNosde) [reply app/Clans.php:28]: That is early return. If the tag passed is null, it returns.

Comment (RoinujNosde) [reply app/Clans.php:28]: Early return avoids nested ifs.

**Pull request merged**